resources/systemd/openhab2.service

```diff
@@ -1,5 +1,5 @@
 [Unit]
-Description=openHAB 2 - empowering the smart home
+Description=openHAB 2 - Empowering the smart home
```

Comment:
I also wanted to do that a few months ago but got scolded by Kai for it.

http://docs.openhab.org/images/logo.png